app/views/devise/passwords/edit.html.erb

```diff
@@ -1,5 +1,6 @@
 <h2>Change your password</h2>
 
+<!--TODO: not sure this is used? can edit password from /users/edit -->
```

Comment:
It was more just a note as I was going through the code and it might be helpful if we come across it later. I feel like it's so low priority that an issue to look into it will never get done, so I guess let it ride?